scenes/vehicle/car_audio_controller.gd

```diff
@@ -17,19 +16,19 @@ extends Node
 @export var recorded_rpm_on := 1000 #rpm on which on_throttle sample is recorded
 @export var recorded_rpm_off := 1000 #rpm on which off_throttle sample is recorded
 @export_range(1, 10, 1) var rpm_interpolation_frames := 5.0#frames to interpolate sound because of rpm spikes
-@export_range(0.0, 10.0, 0.1) var idle_noise_speed := 3.0#how fast the engine fluctuates rpm in idle
-@export_range(0.0, 0.5, 0.01) var idle_noise_octave := 0.5#how much the engine fluctuates rpm in idle (+/- in absolute pitch scale)
-@export var filter_off_throttle = false
-@export var filter_on_throttle = false
-@export_range(0.0, 1000.0, 0.1) var high_pass_cutoff_freq := 0#influence off gas sound per high_pass_filter (0 is no effect,)
-@export_range(10, 22500, 0.1) var low_pass_cutoff_freq := 22500#influence off gas sound per low_pass_filter (22500 is no effect,)
+@export_range(0.0, 10.0, 0.1) var idle_noise_speed := 3.0#how FAST the engine fluctuates rpm in idle
+@export_range(0.0, 0.5, 0.01) var idle_noise_octave := 0.5#how MUCH the engine fluctuates rpm in idle (+/- in absolute pitch scale)
+@export var filter_off_throttle = false#influence off-throttle with filters
+@export var filter_on_throttle = false#influence on-throttle with filters
+@export_range(0.0, 1000.0, 0.1) var high_pass_cutoff_freq := 0#influence off gas sound per high_pass_filter (0 is no effect)
+@export_range(10, 22500, 0.1) var low_pass_cutoff_freq := 22500#influence off gas sound per low_pass_filter (22500 is no effect)
 
 @export_group("Exhaust")
-@export_range(-80.0, 0, 0.1) var db_plopping := -20.0#loudness of exhaust plopping
-@export_range(1.0, 100.0, 0.1) var plopping_frequence := 15.0#
-@export_range(-1.0, 1.0, 0.1) var plopping_probability := 0.7# higher value is more plopping
-@export_range(0.0, 10.0, 0.1) var plopping_time := 1.5# how long exhaust is plopping after release throttle
-@export_range(0.0, 1.0, 0.01) var plopping_rpm_treshold := 0.6#just plopping when release throttle above normalized rpm (set to 0 to plopping on any rpms)
+@export_range(-80.0, 0, 0.1) var db_back_fire := -20.0#db of exhaust back fireing
+@export_range(1.0, 100.0, 0.1) var back_fire_frequence := 15.0#
+@export_range(-1.0, 1.0, 0.1) var back_fire_probability := 0.7# higher value is more back fire
+@export_range(0.0, 10.0, 0.1) var back_fire_time := 1.5#how long exhaust is back fireing after release throttle
+@export_range(0.0, 1.0, 0.01) var back_fire_rpm_treshold := 0.6#just back fire when release throttle above normalized rpm (set to 0 to back-fire on any rpms)
```

Comment:
Could add a space before the # char when commenting. I think its a bit clearer and easier to read if done so.